modules/ocf_www/files/iptables-logrotate

```diff
@@ -0,0 +1,8 @@
+# Logrotate configuration for logging iptables
+/var/log/iptables-outbound.log {
+	weekly
+	missingok
+	rotate 5
+	compress
+	compresscmd xz
```

Comment:
is there a reason we're using xz here when everything else uses gz?

Reply:
I figured we'd get slightly better compression since the logs could blow up. I just tested it and xz was ~30% better than gz which compressed at 11x, although it took a lot more time. I can change it back.